core/app.js

```diff
@@ -90,6 +90,7 @@ app.use(function *(next) {
       this.USE_LAYOUT = 'layout';
     }
   }
+  captionClass = "undefined";
```

Comment:
I'm not quite sure how it would be done removing all global variables. however, by making this global but have it redefined in the app.use (on line 104), it won't ever be overridden as true.